### PR TITLE
feat(Case Overview): explain new migration mode to update Case Overview pages

### PR DESCRIPTION
**Pull request by educhastenier**
Closes [BR-394](https://bonitasoft.atlassian.net/browse/BR-394)

### CODE REVIEW

**Review by pabloalonso — APPROVED** [review]

File: md/migrate-from-an-earlier-version-of-bonita-bpm.md [hunk 1]
```diff
@@ -263,13 +263,46 @@ If you are migrating from a 7.x version:
...
+If you want to update several processes, simply run the command with all the processDefinitionId's one by one.
+
+::: info
+Note: This tool will only change case overview pages. This means that if some of your processes still have process instantiation / task execution forms,
+you need to redesign them in the Studio using Bonita UI designer, as explained in the section above.
+:::
+
+Example of output issued when running the tool:
+
+<script id="asciicast-fvVMGjFTqWsfwY4tmGaOZ7fC7" src="https://asciinema.org/a/fvVMGjFTqWsfwY4tmGaOZ7fC7.js" async></script>
```
Comment (educhastenier) [inline md/migrate-from-an-earlier-version-of-bonita-bpm.md:304]: Maybe I'll find a way to **not** depend on asciinema.org website to host the videocast, but rather host it ourselves, as explained in the asciinema documentation.

Comment (JeremJR) [reply md/migrate-from-an-earlier-version-of-bonita-bpm.md:304]: Yes it's not mandatory as already shown here https://github.com/bonitasoft/bonita-continuous-delivery-doc/pull/130

Comment (educhastenier) [reply md/migrate-from-an-earlier-version-of-bonita-bpm.md:304]: Thx @JeremJR

**Review by danila-m — APPROVED** [review]
LGTM

File: md/migrate-from-an-earlier-version-of-bonita-bpm.md (hunk 1 above)
Comment (educhastenier) [reply md/migrate-from-an-earlier-version-of-bonita-bpm.md:304]: Will be done in another PR very soon.

**Pull request merged**